src/api/routes/address.ts

```diff
@@ -6,8 +6,35 @@ const router = express.Router();
 //GET request /address route
 router.get("/", (req: Request, res: Response, next: NextFunction) => {
     try {
-        console.log(data);
-        res.status(200).json(data);
+        const city: any = req.query.city;
+        const state: any = req.query.state;
+        const country: any = req.query.country;
+        let result = [];
+
+        for (let i = 0; i < data.data.length; i++) {
```

Comment:
Don't use a for loop to iterate over all of them. Use [`.filter()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method.